subprojects/docs/src/samples/java/incubating/modules-multi-project-with-integration-tests/kotlin/buildSrc/src/main/kotlin/myproject.java-conventions.gradle.kts

```diff
@@ -0,0 +1,40 @@
+plugins {
+    java
+}
+
+version = "1.0.2"
+group = "org.gradle.sample"
+
+repositories {
+    mavenCentral()
+}
+
+testing {
+    suites {
+        val test by getting(JvmTestSuite::class) {
+            useJUnitPlatform()
```

Comment:
With https://github.com/gradle/gradle/pull/18308, we'll be able to drop the dependencies block below and just use `useJUnitJupiter("5.7.1")`